gammapy/irf/background.py

```diff
@@ -105,11 +107,20 @@ def from_table(cls, table):
             table, column_prefix="DETY", format="gadf-dl3"
         )
 
+        # TODO: The present HESS and CTA backgroundfits files
+        #  have a reverse order (lon, lat, E) than recommened in GADF(E, lat, lon)
+        #  For now, we suport both.
+
+        data = table[bkg_name].data[0] * data_unit
+        shape = (energy_axis.nbin, fov_lon_axis.nbin, fov_lat_axis.nbin)
+        if np.shape(data) != shape:
```

Comment:
I think we should be even more strict and use `if data.shape == shape[::-1] and not shape == shape[::-1]` or a similar condition. This catches the case, where the transpose of the shape is ambiguous

Reply:
And could emit a warning if this case occurs? Like `log.warning("Assuming transposed axis order for background...")`

Reply:
Yes, I thought of adding a warning - but practically, since this warning will come up in *ALL* HESS and CTA analysis, it might give the user an idea that "gammapy warns a lot, ignore when it cries".

Of course, this is a question of broader philosophy of gammapy - so I have no strong opinion here. Let me know what you decide

Reply:
This could be considered an `info` message than a real `warning`. 
This would alleviate this issue.

Reply:
I would then rather propose the other direction and make it a `log.debug()` statement. The point is just to have at least some possibility of knowing that the data was transposed during reading...

Reply:
Ok! I added `log.debug`

Reply:
Also, an error for shape == shape[::-1